app/templates/web-form.html

```diff
@@ -0,0 +1,207 @@
+{%- extends 'base-' + display_region + '.html' -%}
+
+{%- from 'components/input/_macro.njk' import onsInput -%}
+{%- from 'components/button/_macro.njk' import onsButton -%}
+{%- from 'components/select/_macro.njk' import onsSelect -%}
+{%- from 'components/radios/_macro.njk' import onsRadios -%}
+{%- from 'components/textarea/_macro.njk' import onsTextarea -%}
+{%- from 'components/panel/_macro.njk' import onsPanel -%}
+
+{%- set messages_dict=dict(get_flashed_messages()|groupby('level')) -%}
+{%- set field_messages_dict=dict(messages_dict['ERROR']|groupby('field')) -%}
+
+{%- set form =  {
+    'method': 'POST',
+    'attributes': {
+        'action': url('WebForm:post', display_region=display_region)
+    }
+} -%}
+
+        {%- set country_options =
+            [
+                {
+                    'id': 'england',
+                    'label': {
+                        'text': _('England')
+                    },
+                    'value': 'E'
+                },
+                {
+                    'id': 'wales',
+                    'label': {
+                        'text': _('Wales')
+                    },
+                    'value': 'W'
+                },
+                {
+                    'id': 'northern_ireland',
+                    'label': {
+                        'text': _('Northern Ireland')
+                    },
+                    'value': 'N'
+                }
+            ]
+        -%}
+
+        {%- set query_options =
```

Comment:
not sure why this isn't `category_options` for consistency ?

Reply:
Changed